spec/models/task_spec.rb

```diff
@@ -1,12 +1,10 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  subject(:task) do
-    Task.new(title: "Short text", 
-             description: "Long text", 
-             due_date: Date.today + 1.day )
-  end
-  
+  subject(:employee1) { FactoryGirl.create(:user) }
+  subject(:employee2) { FactoryGirl.create(:user) }
```

Comment:
they are not subjects - it's let - please change

plus it's not the best practice to name vars as foo1, foo2, foo3 - sth more descriptive will be better